uproot/behaviors/TGraphAsymmErrors.py

```diff
@@ -0,0 +1,48 @@
+# BSD 3-Clause License; see https://github.com/scikit-hep/uproot4/blob/main/LICENSE
+
+"""
+This module defines the behaviors of ``TGraphAsymmErrors``, not including ``TGraph``.
+"""
+
+from __future__ import absolute_import
+
+import numpy
+
+import uproot
+
+# dict_keys(['@fUniqueID', '@fBits', 'fName', 'fTitle', 'fLineColor', 'fLineStyle', 'fLineWidth', 'fFillColor', 'fFillStyle', 'fMarkerColor', 'fMarkerStyle', 'fMarkerSize', 'fNpoints', 'fX', 'fY', 'fFunctions', 'fHistogram', 'fMinimum', 'fMaximum', 'fEXlow', 'fEXhigh', 'fEYlow', 'fEYhigh'])
+
+class TGraphAsymmErrors(object):
+    """
+    Behaviors for TGraphAsymmErrors, not including ``TGraph``
+    """
+
+    def values(self):
+        """
+        array[X, Y]
+        """
+        if hasattr(self, "_values"):
+            values = self._values
```

Comment:
Adding arguments to these functions breaks the cache. I think it should just be dropped (also in `errors`). There isn't any processing here that is computationally intensive enough to justify a cache.

Note: if you have a histogram object, it has already been fully read from disk, just due to how ROOT is serialized. These calls to `member` do not invoke a data-read: they're just accessing data in a dict.

Even if these functions return a computed `dstack` or `vstack`, the array would have to contain at least thousands of items before the time to calculate `dstack`/`vstack` is noticeable compared to the time needed for Python to step through the code. It's likely that checking for a cached value costs more than recomputing it.